src/EPR.Payment.Service.Common.UnitTests/Mocks/MockIPaymentRepository.cs

```diff
@@ -105,8 +103,113 @@ public class MockIPaymentRepository
             paymentStatusMock.As<IQueryable<Data.DataModels.Lookups.PaymentStatus>>().Setup(m => m.ElementType).Returns(paymentStatusMockData.ElementType);
             paymentStatusMock.As<IQueryable<Data.DataModels.Lookups.PaymentStatus>>().Setup(m => m.GetEnumerator()).Returns(() => paymentStatusMockData.GetEnumerator());
 
-            // Setup the mock
             return paymentStatusMock;
         }
+
+        public static Mock<DbSet<Common.Data.DataModels.Lookups.RegistrationFees>> GetRegistrationFeesMock()
+        {
+            var registrationFeesMock = new Mock<DbSet<Common.Data.DataModels.Lookups.RegistrationFees>>();
+
+            var registrationFeesData = new List<Common.Data.DataModels.Lookups.RegistrationFees>
+        {
+            // Large Producer - Base Fee (Currently Active)
+            new Common.Data.DataModels.Lookups.RegistrationFees
+            {
+                Group = new Common.Data.DataModels.Lookups.Group { Type = GroupTypeConstants.ProducerType, Description = "large" },
+                SubGroup = new Common.Data.DataModels.Lookups.SubGroup { Description = "large" },
```

Comment:
It seems in the tests we have used it correctly but in the actual code we are using the description field

Reply:
Have changed all the mocks to work properly and match the database seeded data.